### PR TITLE
Initial version of /plan/create API

### PR DESCRIPTION
**Pull request by riecke**
Includes new unit tests.  This has been shown to work through the frontend, but that code will be a separate PR.

### CODE REVIEW

File: src/planscape/plan/views.py [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+
+        # Get the name of the plan.
+        body = json.loads(request.body)
+        name = body.get('name', None)
+        if name is None:
+            raise ValueError("Must specify name")
+
+        # Get the region name
+        region_name = body.get('region_name', None)
+        if region_name is None:
```
Comment (elsiel23) [inline src/planscape/plan/views.py:28]: should we assume this in the long run? if not, can we add a TODO

File: src/planscape/plan/views.py [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+        # Get the geometry of the plan and convert to a MultiPolygon
+        geometry = body.get('geometry', None)
+        if geometry is None:
+            raise ValueError("Must specify geometry")
+        features = geometry.get('features', [])
+        if len(features) > 1 or len(features) == 0:
+            raise ValueError("Must send exactly one feature.")
+        feature = features[0]
+        geom = feature['geometry']
+        if geom['type'] == 'Polygon':
```
Comment (elsiel23) [inline src/planscape/plan/views.py:40]: can we add a comment about why we need to convert this

**Review by elsiel23 — APPROVED** [review]

File: src/planscape/plan/views.py (hunk 1 above)
Comment (riecke) [reply src/planscape/plan/views.py:29]: Added a TODO.

File: src/planscape/plan/views.py (hunk 2 above)
Comment (riecke) [reply src/planscape/plan/views.py:43]: Good point.  Added above.

**Pull request merged**